### DISCUSSION TITLE
FR: New Entry Button Dropdown - List Sections and Selected Section Entry Types

### DISCUSSION THREAD

**Discussion by richhayler** [craftcms/cms#8666] [Ideas] [answered]
It would be great if the "+ new entry" button dropdown (displayed at the top right of the entries listing section - when a specific section is selected in the left-hand "entries" sidebar), displayed entry types for the selected section as well as the other sections themselves.

I'm imagining a "new entry" button with a left dropdown that lists the sections (clicking on one of these would create a new entry for that section) and (if a specific section is selected in the left-hand sidebar) a right dropdown that lists entry types for the selected section (if more than one entry type exists). Clicking on one of the entry types creates a new entry for the selected section and entry type. Clicking the "new entry" button itself would perform the current action - creating a new entry (with the "default" entry type for the selected section). 

If only one entry type exists then no right-hand drop down is displayed and clicking the "new entry" button simply creates a new entry for the selected section.

If the "All Entries" menu option is selected in the left-hand sidebar then the "new entry" button works as the current implmentation with the slight modification that the down arrow (indicating the dropdown) appears on the left so its consistent with the new behaviour described above.

The above proposed design would lessen the number of clicks an author would need to perform to jump straight into creating a new entry with the desired entry type selected.

![1](https://user-images.githubusercontent.com/2505643/57021485-b8da6180-6c6f-11e9-9de8-ee63b203821b.png)

Slightly related to this is #2835

**Comment by richhayler** [comment]
@brandonkelly I just updated the screenshot (in the above post) of the proposed button with some additional notes/explanations for consideration.

**Comment by brandonkelly** [comment, marked as answer]
This was requested previously by #9429, so marking this as a duplicate.
